<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogVPNSettings</class>
 <widget class="QDialog" name="DialogVPNSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VPN Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>VPN Implementation</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="vpn_implementation">
        <item>
         <property name="text">
          <string notr="true">gVisor</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">System</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_10">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string notr="true">MTU</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="vpn_mtu">
        <property name="editable">
         <bool>true</bool>
        </property>
        <item>
         <property name="text">
          <string notr="true">9000</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">1500</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="horizontalGroupBox_2">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QCheckBox" name="vpn_ipv6">
        <property name="text">
         <string>VPN Enable IPv6</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="strict_route">
        <property name="text">
         <string notr="true">Strict Route</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="fake_dns">
        <property name="text">
         <string notr="true">FakeDNS</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="hide_console">
        <property name="text">
         <string>Hide Console</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="single_core">
        <property name="toolTip">
         <string>Add a tun inbound to the profile startup, instead of using two processes.
This needs to be run NekoBox with administrator privileges.</string>
        </property>
        <property name="text">
         <string>Internal Tun</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QGroupBox" name="gb_cidr">
       <property name="title">
        <string>Bypass CIDR</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTextEdit" name="vpn_rule_cidr"/>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="gb_process_name">
       <property name="title">
        <string>Bypass Process Name</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QTextEdit" name="vpn_rule_process"/>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QCheckBox" name="whitelist_mode">
       <property name="toolTip">
        <string>Whether blacklisted or whitelisted, your traffic will be handled by nekobox_core (sing-tun). This is NOT equal to &quot;process mode&quot; of some software.</string>
       </property>
       <property name="text">
        <string>Whitelist mode</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="troubleshooting">
       <property name="text">
        <string>Troubleshooting</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>vpn_implementation</tabstop>
  <tabstop>vpn_mtu</tabstop>
  <tabstop>vpn_ipv6</tabstop>
  <tabstop>strict_route</tabstop>
  <tabstop>fake_dns</tabstop>
  <tabstop>hide_console</tabstop>
  <tabstop>single_core</tabstop>
  <tabstop>vpn_rule_cidr</tabstop>
  <tabstop>vpn_rule_process</tabstop>
  <tabstop>whitelist_mode</tabstop>
  <tabstop>troubleshooting</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogVPNSettings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>399</x>
     <y>575</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogVPNSettings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>399</x>
     <y>575</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
